config/routes.rb

```diff
@@ -113,6 +113,7 @@
   root to: 'signals#index'
   get '/about', to: 'pages#show', id: 'about', as: 'page_about'
   get '/press', to: 'pages#show', id: 'press', as: 'page_press'
+  get '/contact-us', to: 'pages#show', id: 'contact-us', as: 'page_contact_us'
```

Comment:
This is a 404 at the moment, is there a separate ticket / piece of work to create a Contact Us page?

Reply:
No I just forgot to git add the view in. :cry: